### PR TITLE
block apply button in HSR report when GP is not selected and fix mism…

### PR DESCRIPTION
**Pull request by lwyszomi**
…atch in AWW report

@esoergel @czue 

fix for cases: 
http://manage.dimagi.com/default.asp?168229#956708
http://manage.dimagi.com/default.asp?168986#956203

need reset checkpoint in OpmCaseFluff and reindexing

### CODE REVIEW

File: custom/opm/filters.py [hunk 1]
```diff
@@ -120,6 +120,27 @@ def selected(self):
...
+
+    @property
+    def drilldown_map(self):
+        def make_drilldown(hierarchy):
+            list = []
+            for current, next_level in hierarchy.items():
+                if not next_level and not list:
+                    list.append({"val": ALL_OPTION, "text": "All", "next": []})
+                list.append({
+                "val": current,
```
Comment (dimagimon) [inline custom/opm/filters.py]: E122 continuation line missing indentation or outdented

File: custom/opm/filters.py [hunk 2]
```diff
@@ -120,6 +120,27 @@ def selected(self):
...
+                "text": current,
```
Comment (dimagimon) [inline custom/opm/filters.py]: E122 continuation line missing indentation or outdented

File: custom/opm/filters.py [hunk 3]
```diff
@@ -120,6 +120,27 @@ def selected(self):
...
+                "next": make_drilldown(next_level) if next_level else []
```
Comment (dimagimon) [inline custom/opm/filters.py]: E122 continuation line missing indentation or outdented

File: custom/opm/filters.py [hunk 4]
```diff
@@ -120,6 +120,27 @@ def selected(self):
...
+            })
```
Comment (dimagimon) [inline custom/opm/filters.py]: E122 continuation line missing indentation or outdented

**Comment by czue** [comment]
@esoergel do you know if the pillow reset happened?

**Comment by esoergel** [comment]
@czue It didn't, I forgot 'till I saw Anushree's message.  I just kicked it off.

**Pull request merged**